docs/src/pages/pt/getting-started.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Introdução
+---
+
+Astro é um moderno construtor de site estático. Aprenda mais sobre Astro através da [nossa página inicial](https://astro.build/) ou do [nosso post de lançamento](https://astro.build/blog/introducing-astro). Esta página é uma visão geral da documentação do Astro e de todos os recursos relacionados.
+
+Procurando por uma visão rápida do que é Astro? [Visite nossa página inicial.](https://astro.build)
+
+## Experimente Astro
+
+A maneira mais fácil de experimentar Astro é através da execução do comando `npm init astro` em um novo diretório na sua máquina. Nosso assistente para interface de linha de comandos (Wizard CLI) irá te ajudar a iniciar um novo projeto Astro.
+
+Para começar Astro em 5 etapas rápidas e fáceis, visite nosso [Guia de início rápido](quick-start).
+
+Como alternativa, leia nosso [Guia de Instalação](/installation) para uma guia de instalação completo do Astro.
+
+### Experimente Online
+
+Se você tem enteresse em testar Astro online no navegador, você pode usar um editor de código online. Experimente nosso template "Olá Mundo!" em [CodeSandbox](https://codesandbox.io/s/astro-template-hugb3).
```

Comment:
trocar `enteresse` por `interesse`

Reply:
eita, to perdendo meu  português mesmo >.<